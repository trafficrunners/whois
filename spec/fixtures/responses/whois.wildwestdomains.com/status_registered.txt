Domain Name: WILDWESTDOMAINS.COM
Registry Domain ID: 33386184_DOMAIN_COM-VRSN
Registrar WHOIS Server: whois.wildwestdomains.com
Registrar URL: http://www.wildwestdomains.com
Update Date: 2011-11-01T21:31:47Z
Creation Date: 2000-08-22T23:29:11Z
Registrar Registration Expiration Date: 2021-11-01T11:59:59Z
Registrar: Wild West Domains, LLC
Registrar IANA ID: 440
Registrar Abuse Contact Email: abuse@wildwestdomains.com
Registrar Abuse Contact Phone: +1.4806242505
Reseller: WWDomains.com
Domain Status: clientTransferProhibited http://www.icann.org/epp#clientTransferProhibited
Domain Status: clientUpdateProhibited http://www.icann.org/epp#clientUpdateProhibited
Domain Status: clientRenewProhibited http://www.icann.org/epp#clientRenewProhibited
Domain Status: clientDeleteProhibited http://www.icann.org/epp#clientDeleteProhibited
Registry Registrant ID: 
Registrant Name: Registration Private
Registrant Organization: Domains By Proxy, LLC
Registrant Street: DomainsByProxy.com
Registrant Street: 14747 N Northsight Blvd Suite 111, PMB 309
Registrant City: Scottsdale
Registrant State/Province: Arizona
Registrant Postal Code: 85260
Registrant Country: United States
Registrant Phone: +1.4806242599
Registrant Phone Ext: 
Registrant Fax: +1.4806242598
Registrant Fax Ext: 
Registrant Email: WILDWESTDOMAINS.COM@domainsbyproxy.com
Registry Admin ID: 
Admin Name: Registration Private
Admin Organization: Domains By Proxy, LLC
Admin Street: DomainsByProxy.com
Admin Street: 14747 N Northsight Blvd Suite 111, PMB 309
Admin City: Scottsdale
Admin State/Province: Arizona
Admin Postal Code: 85260
Admin Country: United States
Admin Phone: +1.4806242599
Admin Phone Ext: 
Admin Fax: +1.4806242598
Admin Fax Ext: 
Admin Email: WILDWESTDOMAINS.COM@domainsbyproxy.com
Registry Tech ID: 
Tech Name: Registration Private
Tech Organization: Domains By Proxy, LLC
Tech Street: DomainsByProxy.com
Tech Street: 14747 N Northsight Blvd Suite 111, PMB 309
Tech City: Scottsdale
Tech State/Province: Arizona
Tech Postal Code: 85260
Tech Country: United States
Tech Phone: +1.4806242599
Tech Phone Ext: 
Tech Fax: +1.4806242598
Tech Fax Ext: 
Tech Email: WILDWESTDOMAINS.COM@domainsbyproxy.com
Name Server: CNS1.SECURESERVER.NET
Name Server: CNS2.SECURESERVER.NET
Name Server: CNS3.SECURESERVER.NET
DNSSEC: unsigned
URL of the ICANN WHOIS Data Problem Reporting System: http://wdprs.internic.net/
>>> Last update of WHOIS database: 2015-11-30T03:00:00Z <<<

For more information on Whois status codes, please visit https://www.icann.org/resources/pages/epp-status-codes-2014-06-16-en

****************************************************
See Business Registration Listing
****************************************************
Copy and paste the link below to view additional details:
http://who.securepaynet.net/whoischeck.aspx?domain=WILDWESTDOMAINS.COM&prog_id=26domains

The data contained in this Registrar's Whois database, 
while believed by the registrar to be reliable, is provided "as is"
with no guarantee or warranties regarding its accuracy. This information 
is provided for the sole purpose of assisting you in obtaining 
information about domain name registration records. Any use of
this data for any other purpose is expressly forbidden without
the prior written permission of this registrar.  By submitting an
inquiry, you agree to these terms of usage and limitations of warranty.
In particular, you agree not to use this data to allow, enable, or
otherwise make possible, dissemination or collection of this data, in 
part or in its entirety, for any purpose, such as the transmission of 
unsolicited advertising and solicitations of any kind, including spam. 
You further agree not to use this data to enable high volume, automated 
or robotic electronic processes designed to collect or compile this data 
for any purpose, including mining this data for your own personal or 
commercial purposes.

Please note: the owner of the domain name is specified in the "registrant" section. 
In most cases, the Registrar is not the owner of domain names listed in this database.
