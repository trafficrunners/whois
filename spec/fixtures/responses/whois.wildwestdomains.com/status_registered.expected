#domain
  %s == "wildwestdomains.com"

#domain_id
  %s == "33386184_DOMAIN_COM-VRSN"


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2000-08-22 23:29:11 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2011-11-01 21:31:47 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2021-11-01 11:59:59 UTC}


#registrar
  %s %CLASS{registrar}
  %s.id           == "440"
  %s.name         == "Wild West Domains, LLC"
  %s.organization == "Wild West Domains, LLC"
  %s.url          == "http://www.wildwestdomains.com"


#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_REGISTRANT
  %s[0].id            == nil
  %s[0].name          == "Registration Private"
  %s[0].organization  == "Domains By Proxy, LLC"
  %s[0].address       == "DomainsByProxy.com, 14747 N Northsight Blvd Suite 111, PMB 309"
  %s[0].city          == "Scottsdale"
  %s[0].zip           == "85260"
  %s[0].state         == "Arizona"
  %s[0].country       == nil
  %s[0].country_code  == "United States"
  %s[0].phone         == "+1.4806242599"
  %s[0].fax           == "+1.4806242598"
  %s[0].email         == "WILDWESTDOMAINS.COM@domainsbyproxy.com"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == nil
  %s[0].name          == "Registration Private"
  %s[0].organization  == "Domains By Proxy, LLC"
  %s[0].address       == "DomainsByProxy.com, 14747 N Northsight Blvd Suite 111, PMB 309"
  %s[0].city          == "Scottsdale"
  %s[0].zip           == "85260"
  %s[0].state         == "Arizona"
  %s[0].country       == nil
  %s[0].country_code  == "United States"
  %s[0].phone         == "+1.4806242599"
  %s[0].fax           == "+1.4806242598"
  %s[0].email         == "WILDWESTDOMAINS.COM@domainsbyproxy.com"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_TECHNICAL
  %s[0].id            == nil
  %s[0].name          == "Registration Private"
  %s[0].organization  == "Domains By Proxy, LLC"
  %s[0].address       == "DomainsByProxy.com, 14747 N Northsight Blvd Suite 111, PMB 309"
  %s[0].city          == "Scottsdale"
  %s[0].zip           == "85260"
  %s[0].state         == "Arizona"
  %s[0].country       == nil
  %s[0].country_code  == "United States"
  %s[0].phone         == "+1.4806242599"
  %s[0].fax           == "+1.4806242598"
  %s[0].email         == "WILDWESTDOMAINS.COM@domainsbyproxy.com"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil


#nameservers
  %s %CLASS{array}
  %s %SIZE{3}
  %s[0] %CLASS{nameserver}
  %s[0].name == "cns1.secureserver.net"
  %s[1] %CLASS{nameserver}
  %s[1].name == "cns2.secureserver.net"
  %s[2] %CLASS{nameserver}
  %s[2].name == "cns3.secureserver.net"
